modules/ROOT/pages/api-experience-hub-lightning-components.adoc

```diff
@@ -1,57 +1,104 @@
+Each API Experience Hub Lightning web component has different configuration options and adds different functionality to the portal.
 
-Many API Experience Hub Lightning components show data from Anypoint Platform. Portal admins can change some of these data fields, such as API names, descriptions, and icons. API Experience Hub saves these changes but does not push the changes back to Anypoint Platform.
+Many API Experience Hub Lightning web components show data from Anypoint Platform. Portal admins can change some of these data fields, such as API names, descriptions, and icons. API Experience Hub saves these changes but does not push the changes back to Anypoint Platform.
+
+To add API Experience Hub components, click the *Components* lightning bolt icon to display the component palette. You can drag any component from the component palette to the current page. Select the component on the page to show the component’s configuration options. You can edit the values by entering or selecting values in the component property fields. 
+
+To delete a component, click the delete icon and click *Delete* from the dialog.
+
+== API Console
 
-* API Console
-+
 The interactive API Console component shows information about your APIs and their methods. Endpoints are displayed in alphabetical order.
 +
 Console components include API Console, API Console Documentation, API Console Instances, API Console Specification, and API Console Request Panel. In the Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the Summary tab in an API Console results in Summary tab selection in all API Consoles on the same page that display the same API.
 +
 The component shows the public API instances and documentation from Anypoint Exchange.
 
-* Asset Detail Header
-+
+== Asset Detail Header
+
 The Asset Details Header component displays the API header, including icon, name, contact email, and description.
-+
+
 The API icon, name, and description shown are set in the API Experience Hub control panel when you add an API version to your community. The contact email is set in Anypoint Exchange.
-+
-You can specify values for the following parameters of the API Header:
 
-* Asset Version 
-+
+//You can enter or select the following Asset API Header properties:
+//
+//* Record id
+//* Show API Contact
+//* Show API Clients
+//* Show instance URL
+//* Show categories
+//* Show tags
+
+== Asset Version 
+
 Shows the API version from Exchange when only one version is visible.
-* Version State
-+
+
+== Version State
+
 Shows the API version lifecycle state.
 * Asset Documentation Viewer
-+
+
 The API Documentation Viewer component shows an API’s documentation from Anypoint Exchange. The documentation can have multiple pages, and each page can have multiple sections.
 
-* Assets Carousel
+== Assets Carousel
```

Comment:
Is there a way to rewrite this section to be less document-the-UI? Maybe just a screenshot with callouts? Do you need to describe the title and subtitle?

Reply:
I'd like to discuss this more with you after the release.